addons/website/models/website.py

```diff
@@ -241,11 +245,138 @@ def unlink(self):
         attachments_to_unlink.unlink()
         return super(Website, self).unlink()
 
-    def create_and_redirect_to_theme(self):
+    def create_and_redirect_configurator(self):
         self._force()
-        action = self.env.ref('website.theme_install_kanban_action')
+        action = self.env.ref('website.start_configurator_act_url')
```

Comment:
For the record, the correct way of reading/returning an action dict is `env['ir.actions.actions']._for_xml_id(<xmlid>)`.
It takes care of the missing sudo() which would crash if this was ever used by a non-admin user (probably not the case here) Good practice anyway, less brittle, and more idiomatic than `ref() + action.read()[0]` ;)